libevmasm/ExpressionClasses.cpp

```diff
@@ -196,13 +196,13 @@ ExpressionClasses::Id ExpressionClasses::tryToSimplify(Expression const& _expr)
 	if (auto match = rules.findFirstMatch(_expr, *this))
 	{
 		// Debug info
-		//cout << "Simplifying " << *_expr.item << "(";
-		//for (Id arg: _expr.arguments)
-		//	cout << fullDAGToString(arg) << ", ";
-		//cout << ")" << endl;
-		//cout << "with rule " << match->first.toString() << endl;
-		//ExpressionTemplate t(match->second());
-		//cout << "to " << match->second().toString() << endl;
+		/*cout << "Simplifying " << *_expr.item << "(";
```

Comment:
The former way has the benefit that you see that the line is commented out just by looking at the line (in diffs, for example).

If we want to change this, I would prefer an `if`.

Reply:
I suppose an `if` would also make sure that it stays compilable